scapy/contrib/bgp.py

```diff
@@ -8,162 +8,629 @@
 from scapy.packet import *
 from scapy.fields import *
 from scapy.layers.inet import TCP
+from scapy.contrib.bgp_fields import *
 
+def classDispatcher(s,classDict,defClass,index_from=None):
```

Comment:
Please do not use CamelCase / camelBack for functions and variables, and prefer snake_case. For global variables, please use UPPERCASE_SNAKE_CASE.